### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by butalas**
### Description
The branch works the designs of the Login Page. 

### Context
This branch makes changes to the already existing Login Page by implementing relative positioning (removing the absolute styles) using Dimensions. 

### Issue
#66 

### Screenshot
simulator screen shot - iphone x - 2019-02-24 at 16 51 49

### CODE REVIEW

File: screens/Login.js [hunk 1]
```diff
@@ -1,126 +1,93 @@
...
-        <View style={styles.submitButton} />
-        <Text style={styles.submitText}>Submit!</Text>
-        <Text style={styles.forgotText}>Forgot Password?</Text>
-      </View>
+      <ImageBackground style={styles.container}
+          source={require('./../assets/images/background.png')}>
+            <View style={{justifyContent: 'center', alignItems: 'center'}}>
+              <Text style={styles.loginText}>Login</Text>
+              <View style={styles.loginBox}>
+                  <Text style={styles.usernameText}>Username</Text>
```
Comment (ijemmao) [inline screens/Login.js]: This needs to be changed to `TextInput`. Start thinking about practical functionality

File: screens/Login.js [hunk 2]
```diff
@@ -1,126 +1,93 @@
...
+                  <View style={styles.answerLine} />
+                  <Text style={styles.passwordText}>Password</Text>
```
Comment (ijemmao) [inline screens/Login.js]: This needs to be changed to `TextInput`. Start thinking about practical functionality

**Comment by butalas** [comment]
I just added the `TextInput` component so a user can type in their username and password. Here's a screenshot with some text typed into each: 

simulator screen shot - iphone x - 2019-02-24 at 23 01 40

**Comment by butalas** [comment]
Hey! I just updated the file to work with the `OrangeBackground` component Archita made. It looks exactly the same as before.

**Pull request merged**